components/camera/transformpipeline/detector.go

```diff
@@ -27,7 +26,7 @@ type detectorConfig struct {
 
 // detectorSource takes an image from the camera, and overlays the detections from the detector.
 type detectorSource struct {
-	stream       gostream.VideoStream
+	src          camera.VideoSource
```

Comment:
why not camera?

Reply:
https://github.com/viamrobotics/rdk/pull/4487#discussion_r1842930728